### DISCUSSION TITLE
track_target() and wait time

### DISCUSSION THREAD

**Discussion by jeagh** [pybricks/support#1023] [Q&A]
This something that I've never really understood.

Using the following example, I'd like both motors to start at the same time and then run for x and y degrees. The motors won't run unless a wait time is included. I need the shortest possible wait time `and` from experimentation I've found that half a second seems to be the minimum.

Is what I'm doing correct, and if it is, what's the minimum wait time?

`
x_motor.track_target(x)
#wait(500)
y_motor.track_target(y)
wait(500)
`
I'm not sure why the formatting's not working, it usually does.

**Comment by laurensvalk** [comment]
I think you might want to use `run_target` with `wait=False`. See [this example](https://docs.pybricks.com/en/latest/pupdevices/motor.html#waiting-for-two-parallel-actions-to-complete)

`track_target` just starts tracking the new position immediately and skips the acceleration/speed phase. It doesn't wait for anything to complete, so if there is nothing else in your program, it ends right away and it will appear that nothing happened.

**Reply by jeagh** [reply]
Thank you again Lauren,

I knew that I could use wait=False with run_angle() but I didn't think wait=False was an option with run_target. I think while not done is the answer. More experimenting required.
